### ISSUE TITLE
Does not run with CirCuit Python 7.0

### ISSUE THREAD

**Issue by mattpackwood** [adafruit/Adafruit_CircuitPython_PyBadger#52]
I get the error:

```
Traceback (most recent call last):
  File "code.py", line 4, in <module>
  File "adafruit_pybadger/__init__.py", line 12, in <module>
  File "adafruit_pybadger/clue.py", line 32, in <module>
ImportError: no module named 'gamepad'
```

Matt

**Comment by FoamyGuy** [comment]
this should be resolved by #48 which has been merged and a release made now. @mattpackwood when you have a chance to try it again it should be working with 7.0.0 now. You can get the updated library from the github release page, or after 1 day it will get updated in the CircuitPython bundle zip if you use that to load your libraries.

**Comment by mattpackwood** [comment]
Yep, that worked!
